spec/beaker-pe/install/pe_utils_spec.rb

```diff
@@ -860,10 +860,11 @@ def slice_installer_options(host)
 
     it 'sets puppet-agent acceptable_exit_codes correctly for config helper on upgrade' do
       hosts = make_hosts({
-                           :previous_pe_ver => '3.0',
-                           :pe_ver => '4.0',
-                           :roles => ['agent'],
-                         }, 2)
+        :previous_pe_ver => '3.0',
```

Comment:
@jpartlow I'm curious about `:previous_pe_ver` (I see that you added it in these two commits: [1](https://github.com/puppetlabs/beaker-pe/commit/7ef0347dcde6186396681fcec0ba16dae620ff0a), [2](https://github.com/puppetlabs/beaker-pe/commit/0f72aaabce3f496f31f31dbba7a956eff62e6a77)).

I believe before we used `:pe_ver` as the previous version, and `:pe_upgrade_ver` as the upgrade-to version. It looks like you guys would rather go from `:previous_pe_ver` to `:pe_ver`, is that correct?

Adding `:pe_upgrade_ver` in made this spec test go green. I don't think that's really the right solution here, it just looks like a good catalyst for this conversation. I'm wondering if there's a cutoff for when we should use the different schemes, or if there's a version when we know that using `:previous_pe_ver` started working, so we can at least document that.